### ISSUE TITLE
Volume issues & Regular ticking sounds on both playback & recorded files

### ISSUE THREAD

**Issue by avierose** [playfultechnology/audio-guestbook#50]
I've just built the project and confirmed that the logic is working correctly for my phone via serial monitor. 

The first issue I'm having is that the playback volume through the handset is very quiet, almost can't hear it in a quiet room. I have tried increasing the value in 
```
sgtl5000_1.volume(0.95)
```
With no noticeable effect.

The second issue I'm having is that there is a regular ticking sound (4-5/second) over the handset speaker which is also reflected in the recorded .wav files. I'm guessing some connection is bridged but I can find any bad solder joints

**Comment by avierose** [comment]
I've solved the ticking sound, it seems like there are some dodgy capacitors on my telephones circuit board, so I've connected all 4 wires from the handset directly to the Teensy+Audio Shield.

The volume of the playback is still far too quiet to be of any use though. I did try to increase the value in:
`mixer.gain(0, 1.0f);`
`mixer.gain(1, 1.0f);`
but that just introduced distortion without any increase in volume.
